openpype/hosts/maya/plugins/load/actions.py

```diff
@@ -93,7 +93,20 @@ class ImportMayaLoader(load.LoaderPlugin):
 
     """
     representations = ["ma", "mb", "obj"]
-    families = ["*"]
+    families = [
+        "model",
+        "pointcache",
+        "proxyAbc",
+        "animation",
+        "mayaAscii",
+        "mayaScene",
+        "setdress",
+        "layout",
+        "camera",
+        "rig",
+        "camerarig",
+        "staticMesh"
+    ]
```

Comment:
Yeah, we export the geometry patches (meshes) and xgen collection to Maya scene. Having the `import` option on this, does not have sense because you'll need to do extra setup to make it work.

Reply:
True, we could but time is running out on this PR so cant wait on other PRs.